scripts/docs_config/glossary.rst

```diff
@@ -0,0 +1,97 @@
+Glossary
+==========================================================
+
+Homogeneous Memory
+  A collection of memory composed of a single memory type, managed by a singular 
+  driver using a uniform approach.
+
+Heterogeneous Memory
+  A set of memory composed of multiple types of memory technologies, each 
+  requiring distinct handling approaches often managed by separate drivers.
+
+Memory Tiering
+  An organization and hierarchy of different types of memory storage within a 
```

Comment:
I would drop 'hierarchy' from the first sentence since it is described in the second one.

Reply:
done